src/common/index.js

```diff
@@ -1,11 +1,5 @@
-const Asset = require('./Asset');
-const Colors = require('./Colors');
 const Deck = require('./Deck');
-const gameNode = require('./GameNode');
 
 module.exports = {
-    Asset,
-    Colors,
-    Deck,
-    gameNode,
-}
+    Deck
```

Comment:
i'm going to restructure this common stuff, since there are like three common/util modules in here